hashicorp/dev-portal
# feat: update `CodeBlock` to use `react-design-system-components` version

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-leah-featapi-docs-code-block-hashicorp.vercel.app) 🔎
- [Asana task](https://app.asana.com/0/1206815131022336/1205367716254048/f) 🎟️

## 🗒️ What

This PR updates all `CodeBlock` instances in dev-portal to use the `@hashicorp/react-design-system-components` version.

## 🤷 Why

So the website has the updated `CodeBlock` version everywhere. The biggest place where this needed to happen was the hcp api-docs.

## 🛠️ How

The biggest place where this needed to happen was the hcp api-docs. It is also updated on the consul install page in the Note for ARM users section.

## 📸 Design Screenshots


Before
<img width="1039" alt="Screenshot 2024-10-21 at 3 14 39 PM" src="https://github.com/user-attachments/assets/99a8585a-34e5-44cc-8f91-87670e8e25b0">
<img width="906" alt="Screenshot 2024-10-21 at 3 14 25 PM" src="https://github.com/user-attachments/assets/29aea38a-faa5-4e80-ba87-1789cdb1fa42">

After
<img width="1038" alt="Screenshot 2024-10-21 at 3 19 19 PM" src="https://github.com/user-attachments/assets/59f247f0-ff8e-4568-ac81-de9550952e2a">
<img width="917" alt="Screenshot 2024-10-21 at 3 19 02 PM" src="https://github.com/user-attachments/assets/efc95134-3ad8-4df5-9638-b74ef1171e8c">


## 🧪 Testing



1. Go to [vault secrets api-docs preview](https://dev-portal-git-leah-featapi-docs-code-block-hashicorp.vercel.app/hcp/api-docs/vault-secrets)
2. Scroll and notice that the right side code blocks are now the updated version
3. Go to [consul install preview](https://dev-portal-git-leah-featapi-docs-code-block-hashicorp.vercel.app/consul/install)
4. Scroll to Note for ARM users section and notice that the code blocks are now the updated version.

## 💭 Anything else?


When updating the code blocks, I noticed that everything in the `path-truncation-aside` file was not being used anywhere. I deleted that file and the import on the index file.
